nvflare/edge/device/android/sdk/models/Models.kt

```diff
@@ -0,0 +1,205 @@
+package com.nvidia.nvflare.sdk.models
```

Comment:
for me the Models in here means web protocol Models, which means JobResponse, ResultResponse, TaskResponse...

these training models we could put them inside training